public/components/trace_analytics/components/traces/trace_table_helpers.tsx

```diff
@@ -115,6 +115,17 @@ export const getTableColumns = (
       'No'
     );
 
+  const renderStatusCodeErrorsField = (item: number) =>
+    item === undefined || item === null ? (
+      '-'
+    ) : item === 2 ? (
```

Comment:
probably add a comment to avoid confusion in the future, https://github.com/opensearch-project/data-prepper/blob/41e3227e8d79c2fd271ecf1aa92d48be807bf53f/docs/schemas/trace-analytics/otel-v1-apm-span-index-template.md#fields and status.code === 2 means Error, 1 means OK, 0 means Unset

Reply:
Added comment